### ISSUE TITLE
SSL certificate verify failed on startproject

### ISSUE THREAD

**Issue by shonin** [heroku/heroku-django-template#79]
`Django 1.11.6`
`Python 3.6.2`
`pip 9.0.1`
`OSX 10.12.6`

```bash
$ django-admin.py startproject --template=https://github.com/heroku/heroku-django-template/archive/master.zip --name=Procfile myApp
CommandError: couldn't download URL https://github.com/heroku/heroku-django-template/archive/master.zip to master.zip: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:748)>
```

I see an issue with the same error message in pip [here](https://github.com/pypa/pip/issues/4205) but I'm currently unsure if it's related. Help is appreciated and I will post a solution if/when I find one.

**Comment by shonin** [comment]
This issue is happening with a lot of folks using Python 3.6 on Sierra right now.  

See [the issue filed with python](https://bugs.python.org/issue29065)

One way to solve this by running the command `/Applications/Python\ 3.6/Install\ Certificates.command` to install certificates. However this didn't work for me as I didn't have Python 3.6 in my applications directory.

Instead, I just uninstalled Python 3.6 completely using this [SO Answer](https://stackoverflow.com/questions/3819449/how-to-uninstall-python-2-7-on-a-mac-os-x-10-6-4/3819829#3819829) but replacing 2.7 for 3.6

Then I reinstalled it with `brew install python3` and then after fixing up the various issues arising with my path and virtualenvs I was able to start over and successfully validate the cert.

**Comment by iZsh** [comment]
on OSX, using macport, installing curl-ca-bundle solves it:

sudo port install curl-ca-bundle

**Comment by purplediane** [comment]
I'm adding this for others maybe in same situation as me. Plus, even though it got fixed, I don't know if the things I did that didn't *initially* work contributed to the success of the final thing that I did. 

I don't have Macports & I'm still in EL Capitan. I added the `pip/pip.conf` file as in this stack overflow: https://stackoverflow.com/questions/25981703/pip-install-fails-with-connection-error-ssl-certificate-verify-failed-certi/46410817#46410817

But that didn't work. Then did `pip install --trusted-host pypi.python.org pip --upgrade` both global and in my virtualenv, which also didn't work. This command was from https://github.com/pypa/pip/issues/4205

Then I did `/Applications/Python\ 3.6/Install\ Certificates.command` from @shonin above. This finally seemed to fix it, YAY! I did not need to uninstall and reinstall Python 3.6 (using 3.6.2).
